setup.py

```diff
@@ -12,6 +12,7 @@
     "include_package_data": True,
     "install_requires": requirements,
     "license": "Apache License 2.0",
+    "license_files" = ("LICENSE"),
```

Comment:
A couple of things:
1. Since this is a dictionary, we need to use `:` instead of `=`.
2. The value of `license_files` should be a tuple or list instead of a string since
```python
assert "foo" == ("foo")
assert "foo" != ("foo",)
```
So,
```suggestion
    "license_files": ["LICENSE"],
```